netbsd/NetBSDProcessList.c

```diff
@@ -40,10 +40,16 @@ static int pageSize;
 static int pageSizeKB;
 
 static char const *freqSysctls[] = {
-   "machdep.est.frequency.current",
-   "machdep.powernow.frequency.current",
+#if defined(__powerpc__)
    "machdep.intrepid.frequency.current",
+#endif
+#if defined(__mips__)
    "machdep.loongson.frequency.current",
+#endif
+#if defined(__i386__) || defined(__x86_64__)
+   "machdep.est.frequency.current",
+   "machdep.powernow.frequency.current",
+#endif
```

Comment:
This artifact seems odd to be part of the PR …

Reply:
@BenBE I think this happened accidentally while trying to resolve rebase conflicts.